mantidimaging/core/rotation/polyfit_correlation.py

```diff
@@ -46,45 +63,37 @@ def find_center(images: ImageStack, progress: Progress) -> tuple[ScalarCoR, Degr
     m = par[0]
     q = par[1]
     LOG.debug(f"m={m}, q={q}")
+
     theta = Degrees(np.rad2deg(np.arctan(0.5 * m)))
     offset = np.round(m * images.height * 0.5 + q) * 0.5
     LOG.info(f"found offset: {-offset} and tilt {theta}")
-    return ScalarCoR(images.h_middle + -offset), theta
-
 
-def _calculate_correlation_error(images, shared_search_range, min_correlation_error, progress):
-    # if the projections are passed in the partial they are copied to every process on every iteration
-    # this makes the multiprocessing significantly slower
-    # so they are copied into a shared array to avoid that copying
-    shared_projections = pu.create_array((2, images.height, images.width))
-    shared_projections.array[0][:] = images.projection(0)
-    shared_projections.array[1][:] = np.fliplr(images.proj180deg.data[0])
+    return ScalarCoR(images.h_middle + -offset), theta
 
-    do_search_partial = ps.create_partial(do_calculate_correlation_err, ps.inplace3, image_width=images.width)
 
-    arrays = [min_correlation_error, shared_search_range, shared_projections]
-    ps.execute(do_search_partial,
-               arrays,
-               num_operations=min_correlation_error.array.shape[0],
-               progress=progress,
-               msg="Finding correlation on row")
+def compute_correlation_error(index: int, min_correlation_error: np.ndarray, params: dict[str, Any]):
+    shared_projections = params['shared_projections'].array
+    shared_search_range = params['shared_search_range'].array
```

Comment:
Second argument is now the list of arrays, so these need to be unpacked like in `rebin.py`